DESCRIPTION

```diff
@@ -41,10 +41,12 @@ Imports:
     vctrs (>= 0.2.3)
 Suggests: 
     covr,
+    readr,
     rmarkdown,
     sodium,
     spelling,
     testthat (>= 2.1.0),
+    vroom,
```

Comment:
Should I really take both of these on in Suggests? I did so at first to see if there was a difference.

Reply:
Yes, for now.